koku/masu/processor/expired_data_remover.py

```diff
@@ -154,4 +157,10 @@ def remove(self, simulate=False, provider_uuid=None, line_items_only=False):
                 removed_data = self._cleaner.purge_expired_line_item(expired_date=expiration_date, simulate=simulate)
             else:
                 removed_data = self._cleaner.purge_expired_report_data(expired_date=expiration_date, simulate=simulate)
+                if not line_items_only:
+                    expiration_date = self._calculate_expiration_date()
+                    CostUsageReportManifest.objects.filter(
+                        provider__type=self._provider, billing_period_start_datetime__lt=expiration_date
```

Comment:
Do we want to do an equal to the expiration date here or a before? The equal would only give billing periods that start on on the expiration date, where as the `<` would give you all billing periods with a start date that come before the expiration date.

Reply:
I believe the __lt suffix does the same thing as `<` @myersCody https://docs.djangoproject.com/en/3.0/topics/db/queries/#filters-can-reference-fields-on-the-model

Reply:
Ohhh gotcha 👍 That is my bad then, completely missed that.